lib/core/info-drawer/info-drawer.component.ts

```diff
@@ -25,6 +25,9 @@ export class InfoDrawerTabComponent {
     @Input()
     label: string = 'Main tab';
 
+    @Input()
```

Comment:
can you add the jsdoc on this input parameter and add it in the documentation? thanks nad maybe also a small test